content/doc/developer/tutorial/prepare.adoc

```diff
@@ -74,7 +65,7 @@ You are advised to use the  https://github.com/stapler/netbeans-stapler-plugin[
 === IntelliJ IDEA
 IntelliJ 7.0 (or later) users can load pom.xml directly from IDE, and you should see all the source code of libraries and Jenkins core all the way to the bottom. Consider installing the IntelliJ IDEA plugin for Stapler to make the development easier.
 
-Alternatively,emtpy create a new Maven project using Create from archetype and Add an Archetype. Select the GroupId and ArtifactId as above and choose RELEASE as version. In the next screen select io.jenkins.plugins as groupID and choose an artifactId (Project name) and Version to your liking. This will automatically create a maven project based on the specified artifact (e.g. empty-plugin).
+Alternatively, empty create a new Maven project using Create from archetype and Add an Archetype. Select the GroupId and ArtifactId as above, and choose RELEASE as version. On the next screen, select io.jenkins.plugins as groupID and choose an artifactId (Project name) and Version to your liking. This will automatically create a maven project based on the specified artifact (for example, empty-plugin).
```

Comment:
I think the word "empty" is misleading here.  The archetype intentionally provides something that is not empty but is a simple example that can be used as a basis for development.
```suggestion
Alternatively, create a new Maven project using **Create from archetype** and **Add an Archetype**. Select the GroupId and ArtifactId as above, and choose RELEASE as version. On the next screen, select io.jenkins.plugins as groupID and choose an artifactId (Project name) and Version to your liking. This will automatically create a maven project based on the specified artifact (for example, empty-plugin).
```